app/commands/update_current_month_datasource_expenses.py

```
@@ -96,6 +96,7 @@ async def store_datasource_expenses(
                 month=month,
                 defaults={
                     "month_expenses": datasource["details"]["cost"],
```

Is it possible that those keys are not defined and cause an error?

---

In theory no, the optscale docs say this field (and the others we use in the script) will always be present. Of course, in practice the API response may be different but given that this value is the whole reason we're running this script, it should fail if the field is missing. So I'll leave it as it is and if it causes any problems down the line we can go back to it